src/main/resources/application.yml

```diff
@@ -13,9 +13,7 @@ jobs:
   enabled: true
   print-requests:
     name: "BatchPrintRequests"
-    cron: "${JOB_PRINT_REQUESTS_CRON:0 0 */1 * * *}" # every hour by default
-#    lock-at-most-for: ${JOBS_PRINT_REQUEST_LOCK_AT_MOST_FOR}
-#    lock-at-least-for: ${JOBS_PRINT_REQUEST_LOCK_AT_LEAST_FOR}
+    cron: "0 0 */1 * * *" # every hour by default
     batchSize: ${JOB_PRINT_REQUESTS_BATCH_SIZE}
```

Comment:
do we want to set a default for this and use spring magic overrides when needed?

Reply:
@NathanRussellValtech just realised i ask the same question here about the batch size 😄